DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/BridgeJavaSdkUtilTests.java

```diff
@@ -118,6 +129,26 @@
     @Mock
     private Call<StudyParticipant> mockGetExternalIdNotMigratedCall;
 
+    @Mock
+    private Call<StudyList> mockGetAllStudies;
```

Comment:
How does mockGetAllStudies get injected?

Reply:
See https://github.com/Sage-Bionetworks/DianUserMigration-Java/pull/12 for how this is used.